src/extension/inpage-script/index.js

```diff
@@ -105,7 +108,7 @@ if (document) {
           }
         });
     });
-  });
+  }, { capture: true });
```

Comment:
what does capture exactly do? 
and do you know what other options are maybe useful? (e.g. `passive`?

Reply:
Added some explanations here:

https://github.com/getAlby/lightning-browser-extension/issues/1307#issuecomment-1232778613

Went through the other options but couldn't find something that makes sense for us.